plugin/documents.py

```diff
@@ -63,6 +69,9 @@ def run() -> None:
 
 class DocumentSyncListener(LSPViewEventListener, AbstractViewListener):
 
+    ACTIONS_ANNOTATION_KEY = "lsp_action_annotations"
+    code_actions_debounce_time = 800
```

Comment:
Probably would make sense to have those timeout defined here to have a good overview